backend/core/models/film.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var filmSchema = new Schema({
+	title: String,
+	originalID: String,
+	img: String,
+	imdbID: String,
+	reviewed: Boolean,
+	details: {
+		director: String,
+		actors: String,
+		description: String,
+		rating: Number,
```

Comment:
I think that's maybe better to rename it to imdbrating (in the case we'll use other ratings in the future)